runtime/neumann/src/lib.rs

```diff
@@ -583,7 +586,81 @@ impl pallet_collator_selection::Config for Runtime {
 	type WeightInfo = ();
 }
 
-/// Configure the pallet template in pallets/template.
+parameter_types! {
+	pub const CouncilMotionDuration: BlockNumber = 5 * DAYS;
```

Comment:
The parameters are good for Turing Network, but long for development and Neumann. We don’t have separate chain_specs for different networks yet, do we?

Reply:
Yes, we need to separate different runtime configurations for different networks.